_posts/en/newsletters/2025-06-20-newsletter.md

```diff
@@ -0,0 +1,127 @@
+---
+title: 'Bitcoin Optech Newsletter #359'
+permalink: /en/newsletters/2025/06/20/
+name: 2025-06-20-newsletter
+slug: 2025-06-20-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Proposal to restrict access to Bitcoin Core Project discussion:**
+  Bryan Bishop [posted][bishop priv] to the Bitcoin-Dev mailing list to
+  suggest that the Bitcoin Core Project limit public access to
+  discussions among its members in order to reduce the amount of
```

Comment:
```suggestion
  discussions among its members to reduce the amount of
```

Reply:
Keeping as-is.  There's a lot of "to"s in this sentence, adding the extra words here makes that a bit jarring.  (Rewriting might be better, but this paragraph has already been revised twice so let's stick with what we have.)